analyzers/src/SonarAnalyzer.CSharp/Rules/RedundancyInConstructorDestructorDeclarationCodeFixProvider.cs

```diff
@@ -148,7 +148,7 @@ private static SyntaxNode RemoveInitializer(SyntaxNode root, ConstructorDeclarat
                 {
                     if (initializer.HasTrailingTrivia)
                     {
-                        newRoot = newRoot.ReplaceNode(ctor, ctor.WithTrailingTrivia(trailingTrivia));
+                        newRoot = newRoot.ReplaceNode(ctor.ParameterList, ctor.ParameterList.WithTrailingTrivia(trailingTrivia));
```

Comment:
could you add a test where these trailing trivia are being used? maybe 

```
    class LambdaCtorTrailing
    {
        private int i;
        LambdaCtor() :                base()            =>               i++             ;   // Noncompliant
    }

```

because from the current test I see that you've fixed what was wrong, but I still don't fully grasp what this is doing